.eslintrc.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    'prettier/prettier': 0,
```

Comment:
Try using this package for fixing this issue:

https://github.com/prettier/eslint-config-prettier